docs/restic.md

```diff
@@ -1,23 +1,33 @@
-including names, commands, etc., may change as we receive feedback and refine our implementation. 
-
-Two new Ark custom resources have been created to support this feature: `PodVolumeBackup` and `PodVolumeRestore`.
-Additionally, a new Ark daemonset has been created that runs two controllers, one for each of the two new CRDs, on
-each node in the cluster. When an Ark backup is created that includes pods annotated for restic backup, the main Ark 
-backup controller will create a `PodVolumeBackup` custom resource that's owned by the `Backup`. The pod volume backup
-controller running on the pod's node will observe the new custom resource, and will run a restic backup of the volume 
-(accessing the volume's data via a hostPath mount of `/var/lib/kubelet/pods`). The main Ark backup controller will 
-wait for the `PodVolumeBackup` to complete before completing the Ark backup. Restores proceed similarly with some 
-minor differences to account for the fact that a new pod/volume is being created.
+As of v0.9.0, Ark has support for backing up and restoring Kubernetes volumes using a free open-source backup tool called 
+[restic][1].
+
+Ark has always allowed you to take snapshots of persistent volumes as part of your backups, provided you’re using one of 
+the supported cloud providers’ block storage offerings (Amazon EBS Volumes, Azure Managed Disks, Google Persistent Disks). 
+And, since v0.6.0, we’ve had a plugin model which enables anyone to easily implement additional object and block storage 
+backends, outside of the main Ark repository.
+
+We integrated restic with Ark so that users have an out-of-the-box solution for backing up and restoring any type of Kubernetes
+volume*. We view this as a new capability for Ark, rather than a replacement for existing functionality. If you're running on AWS, and
+taking EBS snapshots as part of your regular Ark backups, great! There's no need to switch to using restic. However, if you've
+been waiting for a snapshot plugin to be implemented for your storage platform, or you're using EFS, AzureFile, NFS, emptyDir, 
+local, or any other volume type that doesn't have a native snapshot concept, restic may be for you.
+
+Additionally, since restic is not tied to a specific storage platform, this integration paves the way for future work to enable
+cross-volume-type data migrations. Stay tuned as this evolves!
+
+\* hostPath volumes are not supported, but the [new local volume type][4] is!
 
 ## Setup
 
-This setup guide assumes you already have a working Ark v0.8.1+ installation. If not, go [here][2] for instructions.
+This setup guide assumes you already have a working Ark v0.8.1+ installation. If not, go [here][2] for instructions. It
+also assumes you have the [latest release tag][3] of the Ark repo cloned locally.
+
+**NOTE**: If you're upgrading from  a previous pre-release version of Ark with restic, before you proceed you'll need to:
```

Comment:
Is there some markdown that will allow us to delineate this entire NOTE section (horizontal lines maybe)? And then have "1. Download ..." come after. Or maybe have a new section above "## Setup" that calls out what to do if upgrading from a previous alpha?